pandas/tests/indexing/test_multiindex.py

```diff
@@ -230,7 +230,7 @@ def test_iloc_getitem_multiindex(self):
         # corner column
         rs = mi_int.iloc[2, 2]
         with catch_warnings(record=True):
-            xp = mi_int.ix[:, 2].ix[2]
```

Comment:
not that we care too much, but is this case looks slightly different from below (where testing for ``.ix``)?

Reply:
Yes, because as long as first level is non-int, ``.ix`` is fine. Adding a comment.